### ISSUE TITLE
Keep crafting window AND Auctionator window open at same time

### ISSUE THREAD

**Issue by speedweaver** [Auctionator/Auctionator#1315]
Please include as much of the following information as possible to help me fix the bug:

## Are you on the retail or the classic version of WoW?
Retail

## Is this a bug or a feature request?
Not sure. It USED to work this way then stopped in last 6 months

# Bug:
## Brief Description of Bug

We used to be able to open the Auctionator app AND the Skills tab (for example BS or Tailor) so I could literally click the mat name and it would pop in the search bar in Auctionator.  Now, we can only have 1 or the other window open.  So I have to remember ALL the mats and the counts I need.

## Steps to Reproduce

Open Auctionator.  Open your crafting window.  Auctionator closes.

## Loaded Addons

DBM, Healbot, Postal, Molinari, Altoholic, announce_Interrupts, incognito, Weak Auras,

## Back Trace (LUA Error Output)

NA

# Feature Request:
## Is your feature request related to a problem? Please describe
Not sure.  It just stopped working this way about 6-8 months ago.

## Describe the solution you'd like
I would appreciate it if we could keep both the crafting window AND Auctionator up at the same time.

## Describe alternatives you've considered
Basically, only alt is how it works now. 

## Additional context
xx

**Comment by plusmouse** [comment]
This happens because Blizzard made the crafting window too large to fit alongside the AH on most monitors. You can work around the issue with an addon like https://www.curseforge.com/wow/addons/demodal to keep both windows open or you can reduce your UI scale so that everything is smaller.
